src/dos/drive_overlay.cpp

```diff
@@ -833,18 +836,18 @@ bool Overlay_Drive::FindNext(DOS_DTA & dta) {
 
 bool Overlay_Drive::FileUnlink(char * name) {
 //TODO check the basedir for file existence in order if we need to add the file to deleted file list.
-	Bit32u a = GetTicks();
-	if (logoverlay) LOG_MSG("calling unlink on %s",name);
-	char basename[CROSS_LEN];
-	safe_strcpy(basename, basedir);
-	safe_strcat(basename, name);
-	CROSS_FILENAME(basename);
-
-
-	char overlayname[CROSS_LEN];
-	safe_strcpy(overlayname, overlaydir);
-	safe_strcat(overlayname, name);
-	CROSS_FILENAME(overlayname);
+auto a = GetTicks();
+if (logoverlay)
+	LOG_MSG("calling unlink on %s", name);
+char basename[CROSS_LEN];
+safe_strcpy(basename, basedir);
+safe_strcat(basename, name);
+CROSS_FILENAME(basename);
+
+char overlayname[CROSS_LEN];
+safe_strcpy(overlayname, overlaydir);
+safe_strcat(overlayname, name);
+CROSS_FILENAME(overlayname);
 //	char *fullname = dirCache.GetExpandName(newname);
```

Comment:
Something's taken the formatting off course here? - I can't see what though.

Reply:
looks like `format-commit.sh` freaked out a bit, manually fixing.